client/block-builder/src/lib.rs

```diff
@@ -43,7 +43,7 @@ use sr_api::{Core, ApiExt, ApiErrorFor};
 pub use runtime_api::BlockBuilder as BlockBuilderApi;
 
 /// Error when the runtime failed to apply an extrinsic.
-pub struct ApplyExtrinsicFailed(pub sr_primitives::ApplyError);
+pub struct ApplyExtrinsicFailed(pub sr_primitives::InclusionError);
```

Comment:
My logic was that I'd leave this as is to denote that `apply_extrinsic` failed because of the following reason which is represented by `InclusionError`.

I thought maybe it would be better to name `apply_extrinsic` as `include_extrinsic`, but that is getting out of hand...

Reply:
That said, I am not strong on it. Give me a sign (thumbs up will do) and I will make the change